lib/improver/wind_gust_diagnostic.py

```diff
@@ -31,12 +31,13 @@
 """Module containing plugin for WindGustDiagnostic."""
 
 import warnings
-
+import numpy as np
```

Comment:
Where are you using this?

Reply:
Opps I took out the bit that was using that. I'll take out the import.